streaming/streaming.go

```diff
@@ -403,13 +438,20 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		fmt.Printf("flexTemplateRequest: %+v\n", req)
 		return fmt.Errorf("unable to launch template: %v", err)
 	}
-	printDataflowJob(conv, datastreamCfg, respDf, project)
+	printDataflowJob(conv, datastreamCfg, respDf, project, streamingCfg.DataShardId)
 	return nil
 }
 
-func printDataflowJob(conv *internal.Conv, datastreamCfg DatastreamCfg, respDf *dataflowpb.LaunchFlexTemplateResponse, project string) {
+func printDataflowJob(conv *internal.Conv, datastreamCfg DatastreamCfg, respDf *dataflowpb.LaunchFlexTemplateResponse, project string, dataShardId string) {
```

Comment:
nit: It seems odd that a printJob method is making modifications to common maps

Reply:
Renamed to `storeGeneratedResources`.